tests/jni_api.rs

```diff
@@ -452,6 +452,17 @@ fn auto_local_null() {
     assert!(null_obj.is_null());
 }
 
+
+#[test]
+pub fn throw_new() {
+    let env = attach_current_thread();
+
+    env.throw_new("java/lang/RuntimeException", "Test Exception")
+        .unwrap();
+    env.exception_occurred().unwrap();
```

Comment:
Shan't it assert the returned throwable is not `null`? Or use a stronger assertion that it is of the requested type ("java/lang/RuntimeException"), and, possibly, has the requested message?